### PR TITLE
adding log for error response in iiif provider

### PR DESCRIPTION
**Pull request by Jenniferstrej**

### CODE REVIEW

File: provider/iiif.py [hunk 1]
```diff
@@ -8,6 +8,7 @@ def try_endpoint(endpoint, logger):
     try:
         response = requests.head(endpoint)
         if response.status_code != 200:
+            logger.info("Error status code != 200. Status code: " + str(response.status_code) + " for URL: " + endpoint)
```
Comment (giorgiosironi) [inline provider/iiif.py]: I think this can be raised to the `error` level, since it leads to a permanent failure for this run

Comment (giorgiosironi) [inline provider/iiif.py]: would also use the standard format `logger.info("%s...%s", parameter1, parameter2)`

Comment (giorgiosironi) [inline provider/iiif.py]: btw, this tries all the figures before giving an error, which I like as it gives more information back

Comment (Jenniferstrej) [inline provider/iiif.py]: Aggreed.

**Pull request merged**